src/main/scala/global/Configs.scala

```diff
@@ -0,0 +1,72 @@
+package global
+
+
+import com.typesafe.config._
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.hbase.HBaseConfiguration
+import org.slf4j.LoggerFactory
+
+import scala.util.Try
+
+
+object Configs{
+
+  val logger = LoggerFactory.getLogger(getClass)
+
+  val config: Config = ConfigFactory.load()
+
+  val conf: Configuration = HBaseConfiguration.create()
+  Try{config.getString("hadoop.security.authentication")}.map(conf.addResource).getOrElse(conf.set("hadoop.security.authentication","kerberos"))
```

Comment:
Is there not a method to get the config value as an `Option` instead of using `Try`?